### DISCUSSION TITLE
Could not resolve "@tiptap/core" and more with vite resolve.preserveSymplinks

### DISCUSSION THREAD

**Discussion by samit4me** [ueberdosis/tiptap#4294] [General Questions]
Tried to install tiptap into an existing Vue 2 app that is using Vite and experienced a lot of dependency errors when attempting to serve the application.

To replicate, I spun up a standalone Vue 2 app using `npm create vue@legacy` and then installed the tiptap deps as [per instructions](https://tiptap.dev/installation/vue2#2-install-the-dependencies) and everything worked, hmmmm 🤔 

Next, I modified the `vite.config.js` file line by line to emulate our Vue 2 app and it turns out that enabling the option [resolve.preserveSymplinks](https://vitejs.dev/config/shared-options.html#resolve-preservesymlinks) it causes all the deps to fail.

Wondering if anyone had any suggestions on how to get around this issue?

**Comment by jack9ye** [comment]
I have the same problem with you. Im using @tiptap/vue-3 3.22.2 with Electron-Forge + Vite + Vue3. Set [resolve.preserveSymplinks](https://vitejs.dev/config/shared-options.html#resolve-preservesymlinks) to false in vite.index.config.ts fix this.

**Reply by bdbch** [reply]
Can you share the error log here? It sounds like your package manager is using symlinks to install cross-dependencies and vite is not picking them up if you don't preserve symlinks by their legacy bundler.
